src/components/Button/Button.tsx

```diff
@@ -29,7 +29,7 @@ export function Button(props: ButtonProps) {
       `}
       {...rest}
     >
-      <div className="w-fit m-auto flex gap-2">
+      <div className="w-fit m-auto flex gap-2 leading-normal">
         <div className="m-auto">{preIcon}</div>
```

Comment:
@mazmassa I've already tried it and the size of the svg does not depend on the size of the div.

This is the result : 
![image](https://github.com/massalabs/ui-kit/assets/72019005/8652d038-c47f-4e0b-a8aa-7ed6038b049a)

Reply:
I can confirm I have had the same conclusion

Reply:
SVG nightmare! Icon's should take the whole place where they are. In this case the div with the desired size...

BTW, the result is bad also in terms of alignments with the text. Well... needs investigation how we can improve that API. For now let's go like that. Just double check the alignments.

@pivilartisant can we set then at least in icons component the default value to be the one we have in buttons (24px 
X 24px)!? This would avoir us to set the size each use of it. It that makes sense!?

Reply:
I suggest this : 

```suggestion
        <div className="m-auto">
          <IconContext.Provider value={{ className: "w-6 h-6" }}>
            {preIcon}
          </IconContext.Provider>
        </div>
```

Reply:
Cool! Let's do it like that for now! 🎉